docs/source/teams/roles_and_permissions.rst

```diff
@@ -1,20 +1,21 @@
 .. _teams-roles-and-permissions:
 
-Roles and permissions
-=====================
+Roles, groups and permissions
```

Comment:
nit: suggest leaving the title unchanged to match the page name and table of contents.

All the content below here looks 🔥 though!